common-lib/test/cmr/common/test/generics.clj

```diff
@@ -14,8 +14,9 @@
     (cmr.common.util/are3
      [body]
      (let [one (do (cutil/time-execution body))
-           two (do (cutil/time-execution body))]
-       (is (<= (first two) (first one))))
+           two (do (cutil/time-execution body))
+           diff (- (first two) (first one))]
+       (is (<= diff 2) (format "durration was %d > %d" diff 2)))
```

Comment:
Is 2 supposed to be two in (<= diff 2) and at the end: diff 2?

Reply:
yea, I was getting fails in bamboo because the sever for once was to fast, so I made it 2, if it is slower then that, I'll take a look, but this was my attempt to enforce a test based on speed.